x-pack/plugins/security_solution/common/endpoint/data_loaders/index_fleet_server.ts

```diff
@@ -23,7 +23,7 @@ export const enableFleetServerIfNecessary = async (esClient: Client, version: st
     rest_total_hits_as_int: true,
   });
 
-  if (res.hits.total > 0) {
+  if (!res.hits.total) {
```

Comment:
I think this is doing the opposite of what it should now.

```
const a = 0
const b = 1

a > 0
false

b > 0
true

!a
true

!b
false
```
If the line was like this, it would evaluate correctly: 
```suggestion
  if (!!res.hits.total) {
```

```
!!a
false

!!b
true
```

Reply:
that's a good point! sorry for the unintentional bug :)

Reply:
Drive-by comment: I think this is a great opportunity for a test, so that little things like this will get caught by CI instead of eagle-eyed code reviewers. :)